src/core/runner.ts

```diff
@@ -212,8 +212,9 @@ export default class Runner {
        */
       pluginManager.onStep(step);
       traceEnabled && (await pluginManager.start('trace'));
-      // call the step definition
-      await step.callback();
+      // invoke the step callback by extracting to a variable to get better stack trace
+      const cb = step.callback;
```

Comment:
[question]
is the stack trace better because of the `javascript name inference concept`? 

Do you have a little example where the difference can be seen? (if you have, of course, just asking for future references in case future contributors (also us) need to understand the why

Reply:
thanks! interesting detail